WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -424,6 +450,18 @@ - (void)showViewAdminForBlog:(Blog *)blog
     [[UIApplication sharedApplication] openURL:[NSURL URLWithString:dashboardUrl]];
 }
 
+- (void)showRemoveSiteForBlog:(Blog *)blog
+{
+    NSString *model = [[UIDevice currentDevice] localizedModel];
+    NSString *title = [NSString stringWithFormat:NSLocalizedString(@"Are you sure you want to continue?\n All site data will be removed from your %@.", @"Title for the remove site confirmation alert, %@ will be replaced with iPhone/iPad/iPod Touch"), model];
+    self.removeSiteActionSheet = [[UIActionSheet alloc] initWithTitle:title
+                                                             delegate:self
+                                                    cancelButtonTitle:NSLocalizedString(@"Cancel", nil)
+                                               destructiveButtonTitle:NSLocalizedString(@"Remove site", @"Button to remove a site from the app")
+                                                    otherButtonTitles:nil];
+    [self.removeSiteActionSheet showInView:self.view];
```

Comment:
I used that to match @mnt's [suggestion](https://github.com/wordpress-mobile/WordPress-iOS/issues/3635#issuecomment-101069353)

On the iPad settings app, removing an account shows an alert view, so maybe we should do that for iPad?